### PR TITLE
github-release 0.8.0

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

The github-release tool has been abandoned by the original developer since 2017.  Progress has continued on a fork by @meterup, including an important fix to address GitHub's deprecation of authentication through query params https://developer.github.com/changes/2020-02-10-deprecating-auth-through-query-param/

This PR updates to v0.7.5 from the forked version of github-release.

### CODE REVIEW

**Comment by chenrui333** [comment]
@iainbethune change lgtm. can you also create an issue in https://github.com/aktau/github-release repo to let the community know as well soliciting the feedback? Thanks!

**Comment by SMillerDev** [comment]
Is the new fork acknowledged by the author as being the new home for the project?

**Comment by kevinburkemeter** [comment]
I'm happy to keep maintaining the fork, yes, I'm not sure you are going to be able to get the previous author to respond given they haven't had any activity on that repo in years.

**Comment by SMillerDev** [comment]
That's the requirement homebrew has to take over a name from another repo. If you can't get the other project to direct to the new repo this could be added under a different name maybe?

**Comment by kevinburkemeter** [comment]
I didn't realize the author was still active on Github, just inactive on that project. I tried sending them an email linking them to this issue.

**Comment by kevinburke** [comment]
Working with the current maintainer on transferring ownership to github.com/github-release/github-release, will ping here when that's done.

**Comment by kevinburke** [comment]
I haven't pushed the 0.7.5 release to github-release yet, I will try to get to it within a day or so.

**Comment by kevinburke** [comment]
Just pushed, to avoid any confusion about which was the newest between that and the fork or different tags, I pushed 0.8.0 directly so we should probably use that one.

**Review by SMillerDev — APPROVED** [review]

**Comment by iainbethune** [comment]
Thanks @SMillerDev - remove "do not merge"?

**Comment by SMillerDev** [comment]
Can you squash all your commits into one `<name> <version>`

**Comment by iainbethune** [comment]
@SMillerDev done!

**Comment by jonchang** [comment]
Excellent work, @iainbethune and @kevinburke. This should appear in Homebrew in about half an hour.

**Pull request closed without merging**
